interfaces/builtin/wayland_test.go

```diff
@@ -139,10 +120,6 @@ func (s *WaylandInterfaceSuite) TestAppArmorSpecOnClassic(c *C) {
 	c.Assert(spec.AddConnectedPlug(s.iface, s.plug, s.classicSlot), IsNil)
 	c.Assert(spec.SecurityTags(), DeepEquals, []string{"snap.consumer.app"})
 	c.Assert(spec.SnippetForTag("snap.consumer.app"), testutil.Contains, "/run/user/[0-9]*/wayland-[0-9]* rw,")
```

Comment:
Since you used 'owner' above now, it would be nice to adjust this for owner so that if we change the above rule to remove it, this would fail.

Reply:
Done. Sorry I missed that.
Thanks for the patient review!